packages/app-store/ee/routing-forms/components/SingleForm.tsx

```diff
@@ -192,6 +192,10 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
     defaultValues: form,
   });
 
+  useEffect(() => {
```

Comment:
When switching from 1 tab to another, hookForm get's initialized with current value of tRPC formQuery results. If a mutation has just happened, then an invalidation is in progress and hookForm initializes before that invalidation is complete. So, it's data not needs resetting